ci-operator/step-registry/openshift-qe/kube-burner/openshift-qe-kube-burner-chain.yaml

```diff
@@ -0,0 +1,8 @@
+
+chain:
```

Comment:
I'm not sure if we need this chain here, I think we can still use the chain thats in each of the workload folders